Apps/WebClient/src/Server/Services/ConfigurationService.cs

```diff
@@ -47,7 +46,7 @@ public class ConfigurationService : IConfigurationService
         /// <returns>The external configuration data.</returns>
         public Models.ExternalConfiguration GetConfiguration()
         {
-            this.logger.LogTrace($"Getting configuration data... {JsonConvert.SerializeObject(this.config)}");
+            this.logger.LogTrace($"Getting configuration data...");
```

Comment:
any reasons why we don't log the return value any more?

Reply:
I took it out as even at trace I don't think its really valuable and it relied on Newtonsoft which we should be removing references as we find them.